CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # @shopify/shopify-app-template-remix
 
+## Unreleased
```

Comment:
You can have this as todays date because it will be released as soon as we merge.